.github/workflows/Build-Test-Code.yml

```diff
@@ -64,6 +64,5 @@ jobs:
     - run: npm install --global yarn
     - run: cd $GITHUB_WORKSPACE && yarn install
     # Run the build
-    - run: cp $GITHUB_WORKSPACE/config-example.json $GITHUB_WORKSPACE/config.json
```

Comment:
This line needs to be replaced with `cp $GITHUB_WORKSPACE/.env.template $GITHUB_WORKSPACE/.env` like you have on the tests does it not?

Reply:
Since the .env will be included at runtime, not at build, it doesn't need the config here - as far as I'm aware it'll only be needed for tests

Reply:
ok. the reason i asked is because for the config.json the build fails without it.

Reply:
Makes sense - we don't want to include the config at build because then the docker image is permanently locked to that config and we'd also probably accidentally end up publishing API keys which is certified Not A Good Idea™